cmd/pool/auction.go

```diff
@@ -154,6 +156,71 @@ func batchSnapshot(ctx *cli.Context) error {
 	return nil
 }
 
+var batchSnapshotsCommand = cli.Command{
+	Name:      "snapshots",
```

Comment:
nit: could also just use the reuse old CLI command with default args num=1?

Reply:
Good idea, done!